### ISSUE TITLE
Docs - code_verifier in authorization_code

### ISSUE THREAD

**Issue by matiseni51** [jazzband/django-oauth-toolkit#1207]
Hi!

I'm testing from scratch and i was following the documentation to implement Authorization Code grant type but at the moment that i made the request to get the access_token, i get {error: invalid_grant}. (I even extend the grant expire time)

I test different combination between this attributes and i succeeded using `CODE_CHALLENGE` instead of `CODE_VERIFIER`
(django-oauth-toolkit/docs/getting_started.rst - line 308 & 319) - 
_just the value though_, like `code_verifier=${CODE_CHALLENGE}`
![image](https://user-images.githubusercontent.com/21182685/193262292-5b54e209-4b05-41eb-9bc5-d342a8330ef4.png)

Looking at those values, I think make sense:
![image](https://user-images.githubusercontent.com/21182685/193262049-59cf2c5d-944b-463c-8488-b5d0ba05d320.png)

If i'm right, maybe the bigger fix would be changing that flag `-d code_challenger=${CODE_CHALLENGE}`

**Comment by tykling** [comment]
thank you for this!

**Comment by matiseni51** [comment]
Glad to help!
I tried different things base on your comment and i replicated the behavior that you mention, therefore I updated the PR with that flag.

Also, just point out a case that seems curious to me: even though i selected a SHA-2 256 Algorithm when i registered a new app, if i don't add the `code_challenge_method` flag in the authorization call, the issue that i describe before still works when i make the token call (Basically, the Grant object is set in `code_challenge_method=S256 `and i can get the token calling with `code_verifier=${CODE_CHALLENGE}` ).
